src/__tests__/fireEvent-textInput.test.tsx

```diff
@@ -41,6 +41,37 @@ test('should not fire only touch-related events on non-editable TextInput', () =
   expect(onLayout).toHaveBeenCalledWith(layoutEvent);
 });
 
+test('should not fire only touch-related events on non-editable TextInput with nested Text', () => {
```

Comment:
I don't really understand this test case, what difference does it make that there is a nested text? You're still targeting the TextInput through a testID

Reply:
Fixed, now querying nested `Text`